src/domain/collaboration/callout/callout.resolver.subscriptions.ts

```diff
@@ -68,8 +68,8 @@ export class CalloutResolverSubscriptions {
     @CurrentUser() agentInfo: AgentInfo,
     @Args({
       name: 'calloutID',
-      type: () => UUID,
-      description: 'The ID of the Callout to subscribe to.',
+      type: () => UUID_NAMEID,
+      description: 'The ID or NameID of the Callout to subscribe to.',
```

Comment:
The Callout NameID is only unique within the containing context, so this is not safe.

Reply:
Addressed in b1b2a7972257c750d4bedc1796771ad439c216d2